opensearch/services/datasets/datasets.php

```diff
@@ -162,6 +178,19 @@ public function getSearchResultOutput(array $result) {
 			$html .= "</span>";
 		}
 
+		//data types
```

Comment:
I am not sure whether we need this feature or not. I believe that, when a user filters data-set with a data type, the result set should have all the registered data types for any data-set.